common.yml

```diff
@@ -38,12 +33,16 @@
     - name: restart gandalf
       service: name=gandalf-server state=restarted
 
+# Configure all docker nodes in parallel
 - hosts: "{{ hosts_prefix }}-tsuru-docker*"
-  serial: 1
   sudo: yes
   roles:
     - docker_server
 
+# Run this serially due to concurrent ssh issue
```

Comment:
If we're going to comment this then it should be the line above `serial: 1` to make it clear what the difference is and make it more likely that we remove the comment at the same time as the line.